src/components/code-block/index.jsx

```diff
@@ -81,7 +82,7 @@ function HighlightedCodeBlock({ code, lang }) {
 				</Suspense>
 			</pre>
 			{repl && (
-				<a class="repl-link" href={`/repl?code=${encodeURIComponent(source)}`}>
+				<a class="repl-link" href={`/repl?code=${encodeURIComponent(textToBase64(source))}`}>
```

Comment:
Sorry, unrelated but missed in #1162 as we for some reason weren't calling `btoa` on this. Grep didn't find any other `btoa` references so I assumed we were all covered.